syntax = "proto3";

package mediocre.functions.ocr.v1;
option csharp_namespace = "Mediocre.Functions.Ocr.v1";

service Ocr {
  rpc GetText(ImageToTextRequest) returns (ImageToTextResponse);
}

message ImageToTextRequest {
}

message ImageToTextResponse {
}
